completion/zsh.go

```diff
@@ -0,0 +1,12 @@
+package completion
```

Comment:
(I skipped reviewing these shell scripts)

Reply:
Only noteworthy parts are that I had to copy some stuff from cobra for powershell (with comments + permalink citing), and that I tried to do as little as possible in each shell.